### ISSUE TITLE
ReferenceError: document is not defined when using React.renderToString

### ISSUE THREAD

**Issue by esseb** [davidtheclark/react-aria-modal#6]
ReferenceError: document is not defined
    at Object.<anonymous> (/home/esseb/repositories/test/node_modules/react-aria-modal/node_modules/no-scroll/index.js:2:11)

**Comment by davidtheclark** [comment]
Thanks. I do not have experience yet with server-side rendering. Seems like this problem will be solved if `no-scroll` just quits early if there's no `document`. Does that sound right to you? (Some of the other modules might have to do the same.)

**Comment by esseb** [comment]
I think that would work, yes.

It's ok if the component fails when it's *used* on the server. One suggestion I've seen to avoid server rendering issues like this is to do `import { canUseDOM } from 'react/lib/ExecutionEnvironment';` and then check `canUseDOM` in my render function and only print out `<AriaModal ...>` if `canUseDOM` is true.

As long as the server doesn't fail when I import `react-aria-modal` I'm good.

**Comment by esseb** [comment]
Similar issue in react-displace:
```
/home/esseb/repositories/test/node_modules/react-aria-modal/node_modules/react-displace/index.js:7
      var result = document.createElement('div');
                   ^
ReferenceError: document is not defined
    at /home/esseb/repositories/test/node_modules/react-aria-modal/node_modules/react-displace/index.js:7:20
```

**Comment by esseb** [comment]
`if (typeof document === 'undefined') { return; }` at the top of `no-scroll/index.js` and at the top inside `module.exports = function(Content, options) {` in `react-displace/index.js` solves my problem, although I don't know if that's the best way to solve this.

**Comment by davidtheclark** [comment]
Thanks. I hope to be able to take care of this within the next day or two.

**Comment by davidtheclark** [comment]
@esseb : I just published new patch versions of react-displace and no-scroll. Maybe, then, if you reinstall this module the thing will work for you. Can you let me know if it does? Thanks.

**Comment by esseb** [comment]
Just tested it now and it works great. Thanks.

I love this library so far. There aren't many dialog libraries out there with good aria support, and even fewer as clean as this is, style and class wise.

**Comment by davidtheclark** [comment]
Really glad to hear that it's working well for you. I'm always curious whether what works for me has any relevance to others :)
